anchor/models/post.php

```diff
@@ -59,6 +59,15 @@ public static function search($term, $page = 1, $per_page = 10) {
 				Base::table('users.bio as author_bio'),
 				Base::table('users.real_name as author_name')));
 
+		foreach ($posts as $key => $post) {
+			if ($post->data['status'] !== 'published') {
+				unset($posts[$key]);
+			}
+		}
+		if (count($posts) < 1) {
```

Comment:
Why not using $query->count()? And this is repeating the $total, maybe a shorthanded if/else form here to replace the one above?